torchao/quantization/prototype/mixed_precision/scripts/naive_intNwo.py

```diff
@@ -0,0 +1,40 @@
+import torch
+
+from torchao.quantization.quant_primitives import (
+    MappingType,
+    ZeroPointDomain,
+)
+
+def intN_weight_only_asym(group_size=32, n=8):
+    def apply_intN_weight_only_quant_asym(weight):
+        # avoid circular dep
+        from torchao.dtypes import to_affine_quantized
+        mapping_type = MappingType.ASYMMETRIC
+        block_size = (1, group_size)
+        target_dtype = torch.int8
+        quant_min = 0
+        quant_max = 2**n-1
```

Comment:
should target_dtype be `torch.uint8` for this?

Reply:
Fixed this.